### ISSUE TITLE
Unpublish product from website when stocked change to zero in admin

### ISSUE THREAD

**Issue by benhuson** [wp-e-commerce/WP-e-Commerce#2035]
If "Unpublish product from website" if stock is zero is checked in the admin, if you change the stock level to zero and save the product (via the admin), it remains published.

I've had some feedback that if this was checked and you set the stock to zero, you would still expect it to automatically unpublished the product, even though you changed the stock to zero manually in the admin.

I think this makes sense.

**Comment by JeffPyeBrook** [comment]
I'm not sure on this one. If you are manually publishing a product and want the stock to be zero that's what it should do. 

The checkbox could be buried or panel turned off for the current admin user and they might not know to change it.. 

Maybe the answer is a warning or error message that the product settings are conflicting?

**Comment by benhuson** [comment]
@JeffPyeBrook Yep, thinking about it logically, if that checkbox is checked and the quantity changes to zero then it should unpublished (and fire a notification email if that checkbox is checked too).

But I guess if you're an administrator making the changes manually you should be able to update them manually as you suggest.

Maybe if a product is updated to have that checkbox checked and stock zero and status of published, the notification/saved message at the top of the saved page could say "Product updated. This product is set to unpublish when stock is zero, Would you like to unpublished it?"

**Comment by benhuson** [comment]
...maybe a bit of a non-issue. Certainly low priority.

**Comment by JeffPyeBrook** [comment]
This holiday season has brought forward a few admin issues. For example, on the purchase log status page we never show the status of the purchase. Had a couple of cases of people scrolling through orders using the buttons ( which aren't aligned properly) thinking they had to pack and ship something that was an incomplete sale. 

Those next and previous buttons are also problematic, they can point to orders that don't exist.

Maybe we can give some of these admin things some love after the new year
